### PR TITLE
Raza

### PR DESCRIPTION
**Pull request by SYEDRAZA12**
i have done Heartford Automation Please have a look

### CODE REVIEW

**Comment by shakirshuvo** [comment]
Please take care of the conflicts

File: Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public void closeBrowser() {
 
     @Given("I am on Amazon homepage")
     public void i_am_on_amazon_homepage() throws IOException {
-        openBrowser();
+        openBrowser("https://www.amazon.com/");
```
Comment (SYEDRAZA12) [inline Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java:32]: Keep the url with parameter

File: Amazon/target/cucumber-results.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<testsuite errors="0" failures="0" name="io.cucumber.core.plugin.JUnitFormatter" skipped="0" tests="1" time="36.068">
-    <testcase classname="Amazon search functionality check" name="Amazon search field functionality check with valid data" time="35.402">
+<testsuite errors="0" failures="0" name="io.cucumber.core.plugin.JUnitFormatter" skipped="0" tests="1" time="30.997">
```
Comment (SYEDRAZA12) [inline Amazon/target/cucumber-results.xml:2]: keep this

File: Amazon/target/cucumber-results.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+    <testcase classname="Amazon search functionality check" name="Amazon search field functionality check with valid data" time="30.769">
```
Comment (SYEDRAZA12) [inline Amazon/target/cucumber-results.xml:3]: keep this

File: Amazon/target/cucumber-usage.json [hunk 4]
```diff
@@ -5,13 +5,13 @@
       {
         "name": "I am on Amazon homepage",
         "aggregatedDurations": {
-          "median": 10.3666894,
-          "average": 10.3666894
+          "median": 6.171442,
```
Comment (SYEDRAZA12) [inline Amazon/target/cucumber-usage.json:8]: keep

File: Amazon/target/cucumber-usage.json [hunk 5]
```diff
@@ -5,13 +5,13 @@
...
+          "average": 6.171442
```
Comment (SYEDRAZA12) [inline Amazon/target/cucumber-usage.json:9]: keep

File: Amazon/target/cucumber-usage.json [hunk 6]
```diff
@@ -5,13 +5,13 @@
...
         },
         "durations": [
           {
-            "duration": 10.3666894,
-            "location": "file:///C:/Study/Selenium/Selenium/Frameworks/BDDWebAutomationFramework_Team1/Amazon/src/test/java/features/Search.feature:5"
+            "duration": 6.171442,
```
Comment (SYEDRAZA12) [inline Amazon/target/cucumber-usage.json:13]: keep

File: Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java (hunk 1 above)
Comment (shakirshuvo) [inline Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java:32]: the url is needed

**Review by shakirshuvo — APPROVED** [review]

File: Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java (hunk 1 above)
Comment (SYEDRAZA12) [inline Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java:32]: keep

Comment (shakirshuvo) [inline Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java:32]: looks good

**Review by shakirshuvo — APPROVED** [review]

File: Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java (hunk 1 above)
Comment (shakirshuvo) [inline Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java:32]: Let's keep this one

**Review by shakirshuvo — APPROVED** [review]

File: Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java (hunk 1 above)
Comment (shakirshuvo) [reply Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java:32]: looks good

Comment (shakirshuvo) [inline Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java:32]: keep

File: Amazon/target/cucumber-html-report.html [hunk 7]
```diff
@@ -299,7 +299,7 @@
...
...-89b4-0af5308995d7","testCaseStartedId":"826a9a79-d4c1-4c17-a064-fba06c52b607"}},{"testStepStarted":{"timestamp":{"seconds":"1600636856","nanos":515620000},"testStepId":"512e3988-01f4-41f5-a9cc-0b32e6eac50c","testCaseStartedId":"826a9a79-d4c1-4c17-a064-fba06c52b607"}},{"testStepFinished":{"testStepResult":{"status":"PASSED","duration":{"nanos":65015000}},"timestamp":{"seconds":"1600636856","nanos":580635000},"testStepId":"512e3988-01f4-41f5-a9cc-0b32e6eac50c","testCaseStartedId":"826a9a79-d4c1-4c17-a064-fba06c52b607"}},{"testStepStarted":{"timestamp":{"seconds":"1600636856","nanos":587047000},"testStepId":"159a8559-6ae5-4661-8aef-e4451ff1917f","testCaseStartedId":"826a9a79-d4c1-4c17-a064-fba06c52b607"}},{"testStepFinished":{"testStepResult":{"status":"PASSED","duration":{"nanos":317139000}},"timestamp":{"seconds":"1600636856","nanos":904186000},"testStepId":"159a8559-6ae5-4661-8aef-e4451ff1917f","testCaseStartedId":"826a9a79-d4c1-4c17-a064-fba06c52b607"}},{"testCaseFinished":{"timestamp":{"seconds":"1600636856","nanos":905469000},"testCaseStartedId":"826a9a79-d4c1-4c17-a064-fba06c52b607"}},{"testRunFinished":{"success":true,"timestamp":{"seconds":"1600636856","nanos":912339000}}}];
```
Comment (shakirshuvo) [inline Amazon/target/cucumber-html-report.html:302]: keep

**Review by shakirshuvo — APPROVED** [review]

File: Amazon/target/cucumber-results.xml (hunk 3 above)
Comment (shakirshuvo) [inline Amazon/target/cucumber-results.xml:3]: keep

File: Amazon/target/cucumber-results.xml (hunk 2 above)
Comment (shakirshuvo) [inline Amazon/target/cucumber-results.xml:2]: keep

File: Amazon/target/cucumber-usage.json [hunk 8]
```diff
@@ -5,13 +5,13 @@
...
+            "location": "file:///Users/syedraza/IdeaProjects/BDDWebAutomationFramework_Team1/Amazon/src/test/java/features/Search.feature:5"
```
Comment (shakirshuvo) [inline Amazon/target/cucumber-usage.json:14]: keep

**Review by shakirshuvo — APPROVED** [review]

File: Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java (hunk 1 above)
Comment (shakirshuvo) [inline Amazon/src/test/java/stepDefinitions/SearchStepDefinition.java:32]: keep

**Review by shakirshuvo — APPROVED** [review]

File: Amazon/target/cucumber-results.xml [hunk 9]
```diff
@@ -1,6 +1,6 @@
...
         <system-out>
             <![CDATA[Given I am on Amazon homepage...............................................passed
 And I enter "Hand Sanitizer" in search field................................passed
```
Comment (shakirshuvo) [inline Amazon/target/cucumber-results.xml:6]: keep

File: Amazon/target/cucumber-usage.json [hunk 10]
```diff
@@ -5,13 +5,13 @@
...
           }
         ]
       }
```
Comment (shakirshuvo) [inline Amazon/target/cucumber-usage.json:17]: keep

**Review by shakirshuvo — APPROVED** [review]

File: Amazon/target/cucumber-usage.json (hunk 10 above)
Comment (shakirshuvo) [reply Amazon/target/cucumber-usage.json:17]: keep please

Comment (shakirshuvo) [reply Amazon/target/cucumber-usage.json:17]: keep please

**Pull request closed without merging**